.github/workflows/cicd.yaml

```diff
@@ -271,6 +282,35 @@ jobs:
           MAVEN_TOKEN: ${{ secrets.OSSRH_TOKEN }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: download shared libraries
+        if: github.event_name == 'push'
+        uses: actions/download-artifact@v2
+        with:
+          name: shared-library-${{ github.sha }}
+          path: java
```

Comment:
These so libraries will download to the sub-directory of `java`?

Reply:
confirmed, all shared libraries appear in top directory inside jar
ref: https://s01.oss.sonatype.org/content/repositories/snapshots/com/4paradigm/openmldb/openmldb-native/0.4.0-allinone-SNAPSHOT/